### PR TITLE
[SOL-1563] Coupon editing

### PR DESCRIPTION
**Pull request by marjev**
@mjfrey , @vkaracic can you please take a look?

### CODE REVIEW

File: ecommerce/static/js/views/coupon_form_view.js [hunk 1]
```diff
@@ -244,7 +265,8 @@ define([
              * Override default saveSuccess.
              */
             saveSuccess: function (model, response) {
-                this.goTo(response.coupon_id.toString());
+                console.log(response);
```
Comment (vkaracic) [inline ecommerce/static/js/views/coupon_form_view.js]: Do you still need this log?

File: ecommerce/static/js/views/coupon_form_view.js [hunk 2]
```diff
@@ -6,19 +6,23 @@ define([
         'backbone.super',
         'backbone.validation',
         'backbone.stickit',
+        'moment',
```
Comment (vkaracic) [inline ecommerce/static/js/views/coupon_form_view.js]: Doesn't seem to be used in this function.

**Comment by vkaracic** [comment]
Apart from the coupon nits :+1: 
I'll go ahead and log a task in Jira for cleaning up the variable names as discussed.
https://openedx.atlassian.net/browse/SOL-1602 feel free to add more.

**Comment by marjev** [comment]
@mjfrey I updated tests, addressed @vkaracic's comments. Could you take a look to make sure this is ready for the 2nd review?

**Comment by vkaracic** [comment]
LGTM :+1:

**Comment by mjfrey** [comment]
Good.
:+1:

**Pull request merged**